automated-comments/elixir/two-fer/use_guards.md

```diff
@@ -0,0 +1,11 @@
+<!--
+This error is raised when the solution doesn't use guards
+-->
+
+Use [guards][1] to prevent `two_fer/1` from executing code with with
+non-binary inputs.
+
+- Remember, [strings][2] are considered [binary data][2]
+
```

Comment:
I'd like to see an example about how to use guards.

Reply:
added